vault/resource_azure_auth_backend_config_test.go

```diff
@@ -97,6 +102,67 @@ func TestAccAzureAuthBackend_wif(t *testing.T) {
 	})
 }
 
+func TestAccAzureAuthBackendConfig_retryFields(t *testing.T) {
+	backend := acctest.RandomWithPrefix("tf-test-azure")
+
+	resourceType := "vault_azure_auth_backend_config"
+	resourceName := resourceType + ".config"
+
+	resource.Test(t, resource.TestCase{
+		ProtoV5ProviderFactories: testAccProtoV5ProviderFactories(context.Background(), t),
+		PreCheck:                 func() { testutil.TestAccPreCheck(t) },
+		CheckDestroy:             testCheckMountDestroyed(resourceType, consts.MountTypeAzure, consts.FieldBackend),
+		Steps: []resource.TestStep{
+			{
+				// Test with all retry fields set
+				Config: testAccAzureAuthBackendConfig_retryFields(backend, 5, "10s", "120s"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldBackend, backend),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMaxRetries, "5"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldRetryDelay, "10s"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMaxRetryDelay, "2m0s"),
+				),
+			},
+			{
+				// Test updating retry fields
+				Config: testAccAzureAuthBackendConfig_retryFields(backend, 10, "5s", "180s"),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(resourceName, consts.FieldBackend, backend),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMaxRetries, "10"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldRetryDelay, "5s"),
+					resource.TestCheckResourceAttr(resourceName, consts.FieldMaxRetryDelay, "3m0s"),
+				),
+			},
+			{
+				// Test with zero max_retries (should be allowed)
+				Config: testAccAzureAuthBackendConfig_retryFields(backend, 5, "1s", "30s"),
```

Comment:
The comment refers to testing with zero `max_retries`, but the config sets it to 5. Could we align the comment with the test input or update the input to match the comment?